mocks/flask-react-mocks/package.json

```diff
@@ -0,0 +1,44 @@
+{
+  "name": "MicrosoftWebTemplateStudio",
+  "version": "0.1.0",
+  "private": true,
+  "dependencies": {
```

Comment:
probably won't need half of these dependencies look inside of the nodedefault template.json to figure out which dependencies can be removed.

Reply:
'cookie-parser': '^1.4.3', 'debug': '~4.1.1', 'morgan': '^1.9.1', 'express': '^4.16.4', 'body-parser': '^1.18.3' @njindal239  these dependencies can be removed.

Reply:
Actually remove everything that isn't the following: "{'bootstrap': '^4.3.1', 'classnames': '^2.2.6', 'react': '^16.8.4', 'react-dom': '^16.8.4', 'react-router-dom': '^4.3.1', 'react-scripts': '2.1.8'}",

Reply:
I am not sure if we would need `nodemon`. The front-end is booting up using react-scripts. So, I think it is not using nodemon anywhere.

Reply:
Yup you can remove Nodemon as well from the dev deps good call.